src/WebHook/UpdateLineItemDto.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\WebHook;
+
+use DateTimeInterface;
+use JsonSerializable;
+
+class UpdateLineItemDto implements JsonSerializable
+{
+    public const STATUS_ACCEPTED = 'accepted';
+    public const STATUS_ERROR = 'error';
+    public const STATUS_IGNORED = 'ignored';
+
+    public const NAME = 'RemoteDeliveryPublicationFinished';
```

Comment:
This is TAO specific data, it belongs to a webhook specific class.

Reply:
But this is the webhook specific class that we have unless we use the UpdateLineItemService, there is no other one.